src/test/ui/existential_types/generic_duplicate_param_use2.rs

```diff
@@ -0,0 +1,15 @@
+// compile-pass
+#![feature(existential_type)]
+
+fn main() {}
+
+// test that unused generic parameters are ok
+existential type Two<T, U>: 'static;
+
+fn one<T: 'static>(t: T) -> Two<T, T> {
+    t
+}
+
+fn two<T: 'static, U: 'static>(t: T, _: U) -> Two<T, U> {
```

Comment:
What happens if you write this same example, but change the return type of this function to `Two<U, T>`?